README.md

```diff
@@ -2,6 +2,21 @@
 
 The experiences developed by HestiaLabs are targeted for any user willing to better understand their personal data. It takes the form of an easy-to-use web app that locally processes the raw personal data downloaded from various sources (Twitter, Facebook, Uber, etc) and creates meaningful visualizations.
 
+## Setup
+
+1. Clone this repo and name the directory `hestialabs-experiences`. Then clone the repo [hestialabs](https://github.com/hestiaAI/hestialabs) and name the directory `hestialabs`. These two directories should be siblings and share a parent directory.
+
+2. Navigate to `hestialabs` and create symlinks to all the packages in the global `node_modules/` folder (see [npm-link documentation](https://docs.npmjs.com/cli/v8/commands/npm-link)):
+
+```sh
+$ cd ../hestialabs
+$ npm link --workspaces
```

Comment:
I think you need to run `npm install` too. At least that's what made things work for me locally.

Reply:
> As discussed, it would be nice to have the code of the old tests in the repo, even if they don't work in the repo. Keep them somewhere where they're not run during the build.

https://github.com/hestiaAI/hestialabs-experiences/pull/664/commits/023e82783873b7a6e545fa113eaddf211f9184b9

Reply:
> I think you need to run `npm install` too. At least that's what made things work for me locally.

Yes, thank you. I'll fix the instructions

Reply:
> In the package.json files, replace license ISC with UNLICENSED, I don't think we're open source yet.

https://github.com/hestiaAI/hestialabs/commit/85668a12449af02660d0ceafdb01c22580ad8ea7